modules/installation-about-disconnected.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_disconnected/installing-disconnected-preparations.adoc
+
+[id="installation-about-disconnected_{context}"]
+= About disconnected installations
+
+In {product-title} {version}, you can perform an installation that does not
+require an active connection to the internet to obtain software components. You
+complete a disconnected installation on only infrastructure that you provision,
+not infrastructure that the installer provisions, so your platform selection is
+limited.
+// maybe point out that you can follow the bare metal installation rules on supported hardware and link to the matrix
+
+If you choose to perform a disconnected installation on a cloud platform, you
+still require access to its cloud APIs and will still require internet access
+to use your cluster. Depending on your network, you might require less internet
+access for an installation on bare metal hardware or on VMware vSphere.
+
+To complete a disconnected installation, you must create a repository that
+mirrors the contents of the {product-title} registry and contains the
+installation media. You can create this mirror on a bastion host, which can
+access both the internet and your closed network, or by using other methods
+that meet your restrictions.
+
+[IMPORTANT]
+====
+Because disconnected installations are always on user-provisioned infrastructure,
+always complete a standard user-provisioned infrastructure installation before
+you attempt a disconnected installation.
+====
+
+[id="installation-disconnected-limits{context}"]
+== Additional limits
+
+Disconnected clusters have the following additional limitations and restrictions:
+
+* Cluster settings always displays errors because the cluster cannot determine
+if updates are available.
+* The authentication Operator might randomly fail.
+* Because you cannot access the required ImageStreamTags, you cannot use the
+contents of the Developer Catalog even if it contains items.
```

Comment:
You could mirror these too.  We should talk to samples and OLM folks about cleanly workflows for that.  We might want to wave our hands around it here, although patching streams run by the cluster-version-operator or samples-operator requires [additional configuration][1] so your edits don't get stomped.  Once [rhbz#1741391][2] gets fixed, you won't need to patch the CVO-managed image streams anymore, although you will still need to patch any samples-operator-managed image streams (and possibly OLM stuff) because only by-digest pullspecs will fall back to mirrors automatically.

[1]: https://github.com/openshift/release/pull/4585/commits/75de00dca094ad41790187b1902b3d327b5401fd#diff-e9ff14653190666556f490397d19c553R205-R238
[2]: https://bugzilla.redhat.com/show_bug.cgi?id=1741391

Reply:
I'll soften this a little for now and see if more changes need to be made after the OLM disconnected docs land.